<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.0.1 (Python 3.10.6 on linux)" generated="20221118 04:08:40.507" rpa="false" schemaversion="3">
<suite id="s1" name="Register Login Page" source="/home/mahmoud/robot demo2/RobotFrameWorkTestAutomationLevel2/Tests/01__Register_Login_Page.robot">
<test id="s1-t1" name="User Can &quot;Register&quot;" line="13">
<kw name="Begin Web Test" library="Common" type="SETUP">
<kw name="Open Browser" library="SeleniumLibrary">
<arg>about:blank</arg>
<arg>${browser}</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20221118 04:08:40.962" level="INFO">Opening browser 'chrome' to base url 'about:blank'.</msg>
<status status="PASS" starttime="20221118 04:08:40.961" endtime="20221118 04:08:42.495"/>
</kw>
<kw name="Maximize Browser Window" library="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="PASS" starttime="20221118 04:08:42.496" endtime="20221118 04:08:42.603"/>
</kw>
<status status="PASS" starttime="20221118 04:08:40.959" endtime="20221118 04:08:42.604"/>
</kw>
<kw name="&quot;Registeration&quot; Page" library="ecommerceApp">
<kw name="Load" library="Common">
<kw name="Go To" library="SeleniumLibrary">
<arg>${url}</arg>
<doc>Navigates the current browser window to the provided ``url``.</doc>
<msg timestamp="20221118 04:08:42.608" level="INFO">Opening url 'https://magento.softwaretestingboard.com/'</msg>
<status status="PASS" starttime="20221118 04:08:42.607" endtime="20221118 04:08:45.102"/>
</kw>
<status status="PASS" starttime="20221118 04:08:42.606" endtime="20221118 04:08:45.103"/>
</kw>
<kw name="Verify Page Loaded" library="HomePage">
<kw name="Wait Until Page Contains" library="SeleniumLibrary">
<arg>Default welcome msg!</arg>
<doc>Waits until ``text`` appears on the current page.</doc>
<msg timestamp="20221118 04:08:45.326" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=107.0.5304.110)
Stacktrace:
#0 0x555944d1a463 &lt;unknown&gt;
#1 0x555944ade8d8 &lt;unknown&gt;
#2 0x555944abaa50 &lt;unknown&gt;
#3 0x555944b4184d &lt;unknown&gt;
#4 0x555944b558d9 &lt;unknown&gt;
#5 0x555944b3d443 &lt;unknown&gt;
#6 0x555944b0e3c5 &lt;unknown&gt;
#7 0x555944b0f531 &lt;unknown&gt;
#8 0x555944d6cdce &lt;unknown&gt;
#9 0x555944d70192 &lt;unknown&gt;
#10 0x555944d5193e &lt;unknown&gt;
#11 0x555944d71103 &lt;unknown&gt;
#12 0x555944d44d85 &lt;unknown&gt;
#13 0x555944d920a8 &lt;unknown&gt;
#14 0x555944d92239 &lt;unknown&gt;
#15 0x555944dad492 &lt;unknown&gt;
#16 0x7f7c4fcf0b43 &lt;unknown&gt;
</msg>
<msg timestamp="20221118 04:08:45.327" level="FAIL">NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=107.0.5304.110)
Stacktrace:
#0 0x555944d1a463 &lt;unknown&gt;
#1 0x555944ade8d8 &lt;unknown&gt;
#2 0x555944abaa50 &lt;unknown&gt;
#3 0x555944b4184d &lt;unknown&gt;
#4 0x555944b558d9 &lt;unknown&gt;
#5 0x555944b3d443 &lt;unknown&gt;
#6 0x555944b0e3c5 &lt;unknown&gt;
#7 0x555944b0f531 &lt;unknown&gt;
#8 0x555944d6cdce &lt;unknown&gt;
#9 0x555944d70192 &lt;unknown&gt;
#10 0x555944d5193e &lt;unknown&gt;
#11 0x555944d71103 &lt;unknown&gt;
#12 0x555944d44d85 &lt;unknown&gt;
#13 0x555944d920a8 &lt;unknown&gt;
#14 0x555944d92239 &lt;unknown&gt;
#15 0x555944dad492 &lt;unknown&gt;
#16 0x7f7c4fcf0b43 &lt;unknown&gt;
</msg>
<status status="FAIL" starttime="20221118 04:08:45.104" endtime="20221118 04:08:45.336"/>
</kw>
<status status="FAIL" starttime="20221118 04:08:45.103" endtime="20221118 04:08:45.337"/>
</kw>
<kw name="Register" library="HomePage">
<status status="NOT RUN" starttime="20221118 04:08:45.337" endtime="20221118 04:08:45.337"/>
</kw>
<kw name="Fill Personal Details" library="Register">
<status status="NOT RUN" starttime="20221118 04:08:45.338" endtime="20221118 04:08:45.338"/>
</kw>
<status status="FAIL" starttime="20221118 04:08:42.605" endtime="20221118 04:08:45.338"/>
</kw>
<kw name="End Web Test" library="Common" type="TEARDOWN">
<kw name="Close Browser" library="SeleniumLibrary">
<doc>Closes the current browser.</doc>
<status status="PASS" starttime="20221118 04:08:45.340" endtime="20221118 04:08:45.358"/>
</kw>
<status status="PASS" starttime="20221118 04:08:45.339" endtime="20221118 04:08:45.359"/>
</kw>
<tag>functional</tag>
<tag>Register</tag>
<tag>smoke</tag>
<status status="FAIL" starttime="20221118 04:08:40.958" endtime="20221118 04:08:45.359">NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=107.0.5304.110)
Stacktrace:
#0 0x555944d1a463 &lt;unknown&gt;
#1 0x555944ade8d8 &lt;unknown&gt;
#2 0x555944abaa50 &lt;unknown&gt;
#3 0x555944b4184d &lt;unknown&gt;
#4 0x555944b558d9 &lt;unknown&gt;
#5 0x555944b3d443 &lt;unknown&gt;
#6 0x555944b0e3c5 &lt;unknown&gt;
#7 0x555944b0f531 &lt;unknown&gt;
#8 0x555944d6cdce &lt;unknown&gt;
#9 0x555944d70192 &lt;unknown&gt;
#10 0x555944d5193e &lt;unknown&gt;
#11 0x555944d71103 &lt;unknown&gt;
#12 0x555944d44d85 &lt;unknown&gt;
#13 0x555944d920a8 &lt;unknown&gt;
#14 0x555944d92239 &lt;unknown&gt;
#15 0x555944dad492 &lt;unknown&gt;
#16 0x7f7c4fcf0b43 &lt;unknown&gt;
</status>
</test>
<test id="s1-t2" name="User Can &quot;Login&quot;" line="18">
<kw name="Begin Web Test" library="Common" type="SETUP">
<kw name="Open Browser" library="SeleniumLibrary">
<arg>about:blank</arg>
<arg>${browser}</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20221118 04:08:45.363" level="INFO">Opening browser 'chrome' to base url 'about:blank'.</msg>
<status status="PASS" starttime="20221118 04:08:45.362" endtime="20221118 04:08:46.902"/>
</kw>
<kw name="Maximize Browser Window" library="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="PASS" starttime="20221118 04:08:46.903" endtime="20221118 04:08:47.011"/>
</kw>
<status status="PASS" starttime="20221118 04:08:45.361" endtime="20221118 04:08:47.012"/>
</kw>
<kw name="&quot;Login&quot; Page" library="ecommerceApp">
<kw name="Load" library="Common">
<kw name="Go To" library="SeleniumLibrary">
<arg>${url}</arg>
<doc>Navigates the current browser window to the provided ``url``.</doc>
<msg timestamp="20221118 04:08:47.018" level="INFO">Opening url 'https://magento.softwaretestingboard.com/'</msg>
<status status="PASS" starttime="20221118 04:08:47.017" endtime="20221118 04:08:48.845"/>
</kw>
<status status="PASS" starttime="20221118 04:08:47.015" endtime="20221118 04:08:48.845"/>
</kw>
<kw name="Click to Login Link" library="HomePage">
<kw name="Click Link" library="SeleniumLibrary">
<arg>${login_link_variable}</arg>
<doc>Clicks a link identified by ``locator``.</doc>
<msg timestamp="20221118 04:08:48.848" level="INFO">Clicking link 'xpath=//div[@class='panel header']//a[contains(text(),'Sign In')]'.</msg>
<msg timestamp="20221118 04:08:54.767" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5c460&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot</msg>
<msg timestamp="20221118 04:08:54.769" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5cbb0&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot</msg>
<msg timestamp="20221118 04:08:54.771" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5da80&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot</msg>
<msg timestamp="20221118 04:08:54.774" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: HTTPConnectionPool(host='localhost', port=49547): Max retries exceeded with url: /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5dcf0&gt;: Failed to establish a new connection: [Errno 111] Connection refused'))</msg>
<msg timestamp="20221118 04:08:54.775" level="INFO">robot: 2022-11-18 04:08:54 UTC pid: 641976 - Write from robot to dap (_RobotTargetComm) - EXCEPTION - robocorp_ls_core.debug_adapter_core.debug_adapter_threads
Error writing message.

Traceback (most recent call last):
  File "/home/mahmoud/.local/share/JetBrains/IdeaIC2022.2/robotframework-intellij/lib/robotframework_ls/vendored/robocorp_ls_core/debug_adapter_core/debug_adapter_threads.py", line 266, in writer_thread
    stream.flush()
  File "/usr/lib/python3.10/socket.py", line 723, in write
    return self._sock.send(b)
BrokenPipeError: [Errno 32] Broken pipe</msg>
<msg timestamp="20221118 04:08:54.775" level="FAIL">BrokenPipeError: [Errno 32] Broken pipe</msg>
<status status="FAIL" starttime="20221118 04:08:48.846" endtime="20221118 04:08:54.782"/>
</kw>
<kw name="S">
<status status="NOT RUN" starttime="20221118 04:08:54.798" endtime="20221118 04:08:54.799"/>
</kw>
<status status="FAIL" starttime="20221118 04:08:48.846" endtime="20221118 04:08:54.800"/>
</kw>
<kw name="Enter your &quot;Email&quot;" library="Login">
<status status="NOT RUN" starttime="20221118 04:08:54.801" endtime="20221118 04:08:54.801"/>
</kw>
<kw name="Enter your &quot;Password&quot;" library="Login">
<status status="NOT RUN" starttime="20221118 04:08:54.802" endtime="20221118 04:08:54.802"/>
</kw>
<kw name="Submit" library="Login">
<status status="NOT RUN" starttime="20221118 04:08:54.803" endtime="20221118 04:08:54.803"/>
</kw>
<status status="FAIL" starttime="20221118 04:08:47.013" endtime="20221118 04:08:54.804"/>
</kw>
<kw name="End Web Test" library="Common" type="TEARDOWN">
<kw name="Close Browser" library="SeleniumLibrary">
<doc>Closes the current browser.</doc>
<msg timestamp="20221118 04:08:54.809" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecf0106f20&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39</msg>
<msg timestamp="20221118 04:08:54.811" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecf0106d40&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39</msg>
<msg timestamp="20221118 04:08:54.813" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecf0105f30&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39</msg>
<status status="PASS" starttime="20221118 04:08:54.806" endtime="20221118 04:08:54.816"/>
</kw>
<status status="PASS" starttime="20221118 04:08:54.805" endtime="20221118 04:08:54.816"/>
</kw>
<tag>functional</tag>
<tag>Login</tag>
<tag>smoke</tag>
<status status="FAIL" starttime="20221118 04:08:45.360" endtime="20221118 04:08:54.817">BrokenPipeError: [Errno 32] Broken pipe</status>
</test>
<doc>User Can Register and Login</doc>
<status status="FAIL" starttime="20221118 04:08:40.510" endtime="20221118 04:08:54.819"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="2" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="2" skip="0">functional</stat>
<stat pass="0" fail="1" skip="0">Login</stat>
<stat pass="0" fail="1" skip="0">Register</stat>
<stat pass="0" fail="2" skip="0">smoke</stat>
</tag>
<suite>
<stat pass="0" fail="2" skip="0" id="s1" name="Register Login Page">Register Login Page</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20221118 04:08:45.326" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=107.0.5304.110)
Stacktrace:
#0 0x555944d1a463 &lt;unknown&gt;
#1 0x555944ade8d8 &lt;unknown&gt;
#2 0x555944abaa50 &lt;unknown&gt;
#3 0x555944b4184d &lt;unknown&gt;
#4 0x555944b558d9 &lt;unknown&gt;
#5 0x555944b3d443 &lt;unknown&gt;
#6 0x555944b0e3c5 &lt;unknown&gt;
#7 0x555944b0f531 &lt;unknown&gt;
#8 0x555944d6cdce &lt;unknown&gt;
#9 0x555944d70192 &lt;unknown&gt;
#10 0x555944d5193e &lt;unknown&gt;
#11 0x555944d71103 &lt;unknown&gt;
#12 0x555944d44d85 &lt;unknown&gt;
#13 0x555944d920a8 &lt;unknown&gt;
#14 0x555944d92239 &lt;unknown&gt;
#15 0x555944dad492 &lt;unknown&gt;
#16 0x7f7c4fcf0b43 &lt;unknown&gt;
</msg>
<msg timestamp="20221118 04:08:54.767" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5c460&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot</msg>
<msg timestamp="20221118 04:08:54.769" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5cbb0&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot</msg>
<msg timestamp="20221118 04:08:54.771" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5da80&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot</msg>
<msg timestamp="20221118 04:08:54.774" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: HTTPConnectionPool(host='localhost', port=49547): Max retries exceeded with url: /session/3b6572b26a7f603cea77ecfcd257cf39/screenshot (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecebe5dcf0&gt;: Failed to establish a new connection: [Errno 111] Connection refused'))</msg>
<msg timestamp="20221118 04:08:54.809" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecf0106f20&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39</msg>
<msg timestamp="20221118 04:08:54.811" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecf0106d40&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39</msg>
<msg timestamp="20221118 04:08:54.813" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fecf0105f30&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3b6572b26a7f603cea77ecfcd257cf39</msg>
</errors>
</robot>
